src/node_crypto.cc

```diff
@@ -4357,8 +4477,15 @@ void Verify::VerifyFinal(const FunctionCallbackInfo<Value>& args) {
     hbuf = Buffer::Data(args[1]);
   }
 
+  int padding = RSA_PKCS1_PADDING;
+  int saltlen = -2;
+  if (args.Length() >= 4 && args[3]->IsObject()) {
+    verify->GetRSAOptions(env, args[3]->ToObject(), &padding, &saltlen);
```

Comment:
Forgot to comment … can you check the return value here and return from the function if there was an error?